simpeg/simpeg
# Dev

- bug fix in averaging on cyl mesh
- test averaging on cyl mesh
- use the `@property` decorator in the curvi mesh and differential operators as per: https://www.quantifiedcode.com/app/project/933aa3decf444538aa432c8817169b6d?groups=code_patterns%3A3bECxdfc%3Af0&tab=basics). 
- use `format` for strings
- add link-check to docs (and fix broken links) 
- updates to some wildcard imports @grosenkj :) 

This closes #367
